cic-ipv-stub/src/events/callbackFailure.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
Are these json files for testing via sam local or used otherwise in the stub?

Reply:
They're used to test the handlers in the unit tests. I expect you can also use them with sam local.